stonesoup/models/base.py

```diff
@@ -54,6 +54,8 @@ def function(self, state_vector, noise=None, **kwargs):
         """
 
         if noise is None:
+            # noise = sp.zeros(state_vector.shape)
+            # TODO resolve the noise=None generating noise issue
             noise = self.rvs(**kwargs)
```

Comment:
This now contradicts the documentation. Suggest this is addressed in separate pull request.

Reply:
Done.